### DISCUSSION TITLE
[general] The document of `deployment` argument is not correct?

### DISCUSSION THREAD

**Discussion by psychelzh** [ropensci/targets#1208] [General]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

It says: 

> Character of length 1, only relevant to `tar_make_clustermq()` and `tar_make_future()`. If `"worker"`, the target runs on a parallel worker. If `"main"`, the target runs on the host machine / process managing the pipeline.

But according to the manual [here](https://books.ropensci.org/targets/crew.html#heterogeneous-workers), it is also related to `tar_make()` when used with crew package. So, is it time to correct this, given `tar_make_clustermq()` and `tar_make_future()` are superseded.

**Comment by wlandau** [comment]
Thanks, fixed in 8ab431908ea584f990fc2727cd4e2aaabe3d0cb2.
